xml/api/crm_mon-2.33.rng

```diff
@@ -0,0 +1,213 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This should be `crm_mon-2.34.rng`. The main branch is already at API version 2.34.

Reply:
Keep an eye on this before merging. #3338 also introduces an API version 2.34, so whichever gets merged second will need to be 2.35.